lazy-scripts/run_tests_sonarscan.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/sh -l
+
+set -eu
+CMDNAME=$(basename "$0")
+
+OUTPUTDIR=""
+SONAR_ORGANIZATION=""
+SONAR_PROJECT_KEY=""
+REVISION=""
+BRANCH_NAME=""
+PULL_REQUEST_KEY=""
+
+function usage
+{
+  local exitcode=$1
+  cat << EOF >&2
+Usage:
+  $CMDNAME --output-dir --sonar-organization --sonar-project-key --revision {--branch-name|--pull-request-key} [--help]
+  --help                      Show this message and exit
+  -d, --output-dir            (Required) Output directory of coverage files
+  -o, --sonar-organization    (Required) Sonarcloud organization
+  -k, --sonar-project-key     (Required) Project key of the sonarcloud project to record against
+  -r, --revision              (Required) SHA of the commit being analyzed to enable PR/commit annotations.
+  -b, --branch-name           Branch name to associate results. Cannot be used with --pull-request-key
+  -p, --pull-request-key      Pull request key to associate results. Cannot be used with --branch-name
+    The environment variable SONAR_TOKEN is used for the sonar.login property
+EOF
+
+  exit "$exitcode"
+}
+
+function process_cli
```

Comment:
Make the scripts consistent. If you are using cli strategy here, use it in every script

Reply:
I will make everything consistent as we keep adding lazy flow.

Reply:
If you aren't doing now, then create a tech debt ticket to track it.

Reply:
https://usxtech.atlassian.net/browse/LAZY-51